doc-content/jbpm-docs/src/main/asciidoc/Kafka/message-send-event-proc.adoc

```diff
@@ -15,6 +15,12 @@ endif::PAM,DM[]
 ifdef::JBPM,DROOLS,OP[]
 xref:jBPMBPMN2[].
 endif::JBPM,DROOLS,OP[]
+. Select the *Settings* -> *Deployments* -> *Work Item Handlers* tab.
+. Click *Add Work Item Handler*.
+. Enter the following values in the new empty line:
+** *Name*: `Send Task`
+** *Value* : `new org.jbpm.bpmn2.handler.SendTaskHandler()`
```

Comment:
This will work also for signals, but notice that the debug log will be:
Sending message: null
Because it's trying to retrieve the parameter "Message" that only applies to messages.
For signals, the signal information is inside "Data", but currently there's no default workitemhandler that retrieves it, but probably these are minor details (only debug logging is affected)